keyboards/cyboard/dactyl/info.json

```diff
@@ -0,0 +1,744 @@
+{
+    "keyboard_name": "Dactyl",
+    "usb": {
+        "pid": "0x0000",
+        "device_version": "0.0.1"
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "diode_direction": "ROW2COL",
+    "matrix_pins": {
+        "rows": ["GP8", "GP7", "GP28", "GP6", "GP23", "GP27", "GP21"],
+        "cols": ["GP9", "GP26", "GP22", "GP20", "GP29", "GP4", "GP5"]
+    },
+    "split": {
+        "soft_serial_pin": "GP0",
+        "soft_serial_speed": 2
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "rgb_matrix": {
+        "driver": "WS2812"
+    },
+    "ws2812": {
+        "pin": "GP1",
+        "driver": "vendor"
+    },
+    "debounce": 5,
```

Comment:
```suggestion
```

Reply:
ah is this just the default behavior?

Reply:
> ah is this just the default behavior?

correct, default int for all debounce types (`quantum/debounce/`)